src/Wrapper.js

```diff
@@ -295,6 +295,24 @@ export default class Wrapper implements WrapperInterface {
   }
 
   /**
+   * Sets vm methods
+   *
+   * @param {Object} methods - methods to set
+   */
+  setMethods(methods: Object) {
```

Comment:
Can you add this method to the flow interface - https://github.com/eddyerburgh/avoriaz/blob/master/flow/wrapper.flow.js